src/cunit/ut_cunit.c

```diff
@@ -83,6 +99,22 @@ static TestMode_t get_test_mode( void )
     return gTestMode;
 }
 
+void UT_Manage_Suite_Activation(int groupID, bool enable_disable)
+{
+    if (enable_disable == false && gGroupFlag.disable_count < MAX_OPTIONS)
+    {
+        gGroupFlag.disable_value[gGroupFlag.disable_count++] = groupID;
+    }
+    else if (enable_disable == true && gGroupFlag.enable_count < MAX_OPTIONS)
+    {
+        gGroupFlag.enable_value[gGroupFlag.enable_count++] = groupID;
+    }
+    else
+    {
+        UT_LOG_ERROR("Too many -d or -e options\n");
```

Comment:
this can never happen.

If ( enable_disable == false )
{
.. do stuff
return
}

true case

do stuff

return